components/radiogroup/overview.md

```diff
@@ -53,58 +55,61 @@ Chosen gender: @( ChosenGender == 0 ? "no selection yet" : ChosenGender.ToString
 
-## Features
+## Layout
 
-The RadioGroup provides the following features:
+The RadioGroup component provides two ways to render the list of options - in a *vertical* or in a *horizontal* fashion. [Read more about the Blazor RadioGroup layouts]({%slug radiogroup-layout%}).
 
-* `Class` - the CSS class that will be rendered on the main wrapping element of the component.
+## Label Position
 
-* `Data` - allows you to provide the data source. Required.
+The RadioGroup component provides two ways to render the labels of the radio buttons - *before* or *after* the buttons themselves. [Read more about the Blazor RadioGroup label position]({%slug radiogroup-label-position%}).
 
-* `Enabled` - whether the component is enabled.
+## Appearance Settings
 
-* `Id` - renders as the id attribute on the main wrapping `ul` element.
+The Blazor RadioGroup has settings for customizing the button appearance. [Read more about the Blazor RadioGroup settings]({%slug radiogroup-appearance%}).
 
-* `LabelPosition` - whether the labels render after or before the radio button itself.
+## Events
 
-* `Layout` - whether the buttons are rendered vertically or horizontally.
+The Blazor RadioGroup fires value change and blur events that you can handle and further customize its behavior. [Read more about the Blazor RadioGroup events]({%slug radiogroup-events%}).
 
-* `Name` - lets you choose your own `name` attribute for the underying `<input type=radio>` elements.
+## RadioGroup Parameters
 
-* `TItem` - the type of the model to which the component is bound. Required if you can't provide `Data` or `Value`. Determines the type of the reference object.
+The Blazor RadioGroup provides various parameters to configure the component. Also check the [RadioGroup public API](https://docs.telerik.com/blazor-ui/api/Telerik.Blazor.Components.TelerikRadioGroup-2).
 
-* `TValue` - the type of the value field from the model to which the component is bound. Required if you can't provide `Data` or `Value`. Determines the type of the reference object.
-
-<!-- * `TabIndex` - the `tabindex` attribute rendered on the dropdown. -->
-
-* `TextField` - the name of the field from the model that will be shown to the user. Defaults to `Text`.
-
-* `ValueField` - the name of the field from the model that will populate the underlying `Value`. Defaults to `Value`.
-
-* `Value` and `bind-Value`- get/set the value of the component, can be used for binding. If you set it to a value allowed by the model class value field, the corresponding item from the data collection will be pre-selected. Use the `@bind-Value` syntax for two-way binding, for example, to a variable of your own.
+<style>
+    article style + table {
+        table-layout: auto;
+        word-break: normal;
+    }
+</style>
+| Parameter | Type and Default Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | The CSS class that will be rendered on the main wrapping `<ul class="k-radio-list">` element. |
```

Comment:
Using inline code as adjective will make it hard to read. Follow the same explanation structure on all rows - either start with a verb, or a noun.

```suggestion
| `Class` | `string` | Renders a custom CSS class for the main wrapping element, which is `<ul class="k-radio-list">`. |
```